### DISCUSSION TITLE
Polling interval question

### DISCUSSION THREAD

**Discussion by baba-joe** [tbnobody/OpenDTU#821] [Q&A]
I get frequently data age >60s when a individual inverter is selected in live view, no matter how I set the polling interval?  What is the polling interval setting good for then?
![data_age](https://user-images.githubusercontent.com/130974809/233049027-009f7666-b90f-4dc7-99c9-078efe696d17.png)

**Comment by tbnobody** [comment]
If you have a bad connection, the inverter is polled every 5 seconds, but if no data is exchanged, the data age will increase...

**Reply by baba-joe** [reply]
Not sure what you mean? The DTU is outside right next to the inverters with nothing in between (max. 5m air) and send power is set to second highest. Also I have the version with the external antenna. I have managed to implement all inverters into my Victron setup just now (henne49 script) and its very slow in refreshing data compared to my Victron hardware. I assumed that the polling intervall in DTU settings defines the "resolution" of information I can retrieve from the inverters?

**Reply by DPO99** [reply]
It sounds a little bit crazy but maybe you should not use the maximum PA-Level. Maybe a problem with the power supply?
